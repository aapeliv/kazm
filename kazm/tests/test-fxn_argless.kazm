//calling other func in main 

void other_func() {
  println("In other func");
}

int main() {
  println("About to call other_func()");
  other_func();
  println("Done.");
  return 0;
}
