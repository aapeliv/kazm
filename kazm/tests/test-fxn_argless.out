About to call other_func()
In other func
Done.
